src/__tests__/Storyshots.test.tsx

```diff
@@ -1,10 +1,87 @@
 import initStoryshots from "@storybook/addon-storyshots"
+import { mount } from "enzyme"
+import * as React from "react"
+
+class ErrorBoundary extends React.Component<{ onError: (error) => void }> {
+  state = {
+    errorOccurred: false,
+  }
+
+  componentDidCatch(error, info) {
+    this.setState({ errorOccurred: true }, () => {
+      this.props.onError(error)
+    })
+  }
+
+  render() {
+    if (this.state.errorOccurred) {
+      return null
+    }
+    return this.props.children
+  }
+}
 
 initStoryshots({
   storyNameRegex: /Gene Follow/,
+  /**
+   * TODO:
+   *
+   * - API:
+   *   - Let story specify a sentinel selector to look for to exist.
+   *   - Let story specify a sentinel selector to look for to NOT exist.
+   *   - Maybe make it
+   *
+   * - Need to call tree.unmount() after run, but consider case where test times
+   *   out as well, so this should really run in `afterEach`. However, the
+   *   current option to specify before/after filters through storyshots is that
+   *   they are global, so no good way to scope e.g. `tree` to specific tests.
```

Comment:
We could probably add a timer outside the the recursive wait and check for that manually?

Reply:
Yeah definitely, we should do that regardless when making this into an enzyme extension. It’s just that we may run the risk of the global test framework timeout kicking in earlier and thus the next test starting with our async code still running. But we can figure that out later 👍